docs/contributors-guide.rst

```diff
@@ -104,3 +104,29 @@ Also, to install githooks to format the code per-commit, use:
 .. code-block:: bash
 
    ./.githooks/install
+
+Example
```

Comment:
rendered content is not what we expect. 

![image](https://github.com/ROCm/hipTensor/assets/13180863/1ef5e86f-8c9d-4e46-9ad2-576513f35d92)